dev-libs/libxmlb/libxmlb-0.3.22.recipe

```diff
@@ -0,0 +1,103 @@
+SUMMARY="A library to help create and query binary XML blobs"
+DESCRIPTION="libxmlb is a small helper library that is used to create and \
+query the binary XML-like blobs that are used by AppStream."
+HOMEPAGE="https://github.com/hughsie/libxmlb"
+COPYRIGHT="Richard Hughes"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+
+SOURCE_URI="https://github.com/hughsie/libxmlb/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="103684ed37a45d0aed8f95e97294ed26945b5aeebf44734f3994081eecebb11c"
+SOURCE_FILENAME="libxmlb-$portVersion.tar.gz"
+SOURCE_DIR="libxmlb-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="2.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
+
+PROVIDES="
+	libxmlb$secondaryArchSuffix = $portVersion
+	cmd:xb_tool$secondaryArchSuffix
+	lib:libxmlb$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libzstd$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libxmlb${secondaryArchSuffix}_devel = $portVersion
+	devel:libxmlb$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES_devel="
+	libxmlb$secondaryArchSuffix == $portVersion base
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libzstd$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	lib:libgirepository_1.0
+	gobject_introspection$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libzstd$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	lib:libgirepository_1.0
+	gobject_introspection$secondaryArchSuffix
+	gobject_introspection${secondaryArchSuffix}_devel
+	"
+
+BUILD_PREREQUIRES="
+	lib:libgirepository_1.0
+	gobject_introspection$secondaryArchSuffix
+	gobject_introspection${secondaryArchSuffix}_devel
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:meson
+	cmd:ninja
+	cmd:cmake
+	cmd:diff
+	cmd:g_ir_annotation_tool
+	cmd:g_ir_compiler
+	cmd:g_ir_generate
+	cmd:g_ir_inspect
+	cmd:g_ir_scanner
```

Comment:
Not required, can be removed.

Then sort alphabetically.

Reply:
@icarito in case you still have doubts or questions, don't be afraid to ask. We're here to help. :)